tests/js/spec/views/dashboardsV2/widgetQueries.spec.jsx

```diff
@@ -696,6 +696,68 @@ describe('Dashboards > WidgetQueries', function () {
     );
   });
 
+  it('does not re-query events and sets name in widgets', async function () {
```

Comment:
I was trying to test this in `widgetBuilder.spec.tsx` but I was having trouble selecting the text from inside the chart component so I went with a `widgetQueries` test that re-renders the same component with a different widget query alias to simulate the Legend Alias update scenario.

I think we mock out echarts in `tests/js/setup.ts` and that's why I wasn't able to select the text